content/lessons/functions-and-scope/scope.md

```diff
@@ -8,16 +8,13 @@ tags:
   - Scope
   - JS Fundamentals
   - Function and Block Scope
-description: 'In the previous section we learned about the dynamic context feature of
-JavaScript functions. Today we will explore the static _scope_ and the
-differences of scope vs context. We will demonstrate how scope _work_ directly
-by exploring some of the lesser known gotchas in JavaScript like "hoisting".'
+description: 'We will begin by exploring the three types of _scope_ in JavaScript. We will demonstrate how variable types and scope change the information available to us and learn some of the lesser-known gotchas in JavaScript like "hoisting".'
 timeToCompletion: ~1 hour
-videoLinks: 
+videoLinks:
   - https://www.youtube.com/embed/sjyJBL5fkp8
 preReadQuizLink: https://docs.google.com/forms/d/e/1FAIpQLSeM4X1hctc26jxQr9-5gT3a9WOHpLy_6I2CymoUg1uGz4hOCw/viewform
-readingLinks: 
+readingLinks:
   - link: https://hackernoon.com/understanding-javascript-scope-1d4a74adcdf5
     description: Scope is a powerful concept in JS - let's deepen our understanding.
     title: Understanding JavaScript Scope
----     
\ No newline at end of file
+---
```

Comment:
Don't know if you meant to change the four dashes here to `+---`; does this change any behavior on the site?